src/csvcubed/utils/sparql_handler/data_cube_inspector.py

```diff
@@ -30,6 +35,37 @@
 )
 
 
+def _figure_out_end_user_column_type(
```

Comment:
Can we stick this down the bottom of the module? And perhaps call it map component to end user column type?